src/app/components/Tokens/TokenList.tsx

```diff
@@ -129,7 +130,7 @@ export const TokenList = (props: TokensProps) => {
           align: TableCellAlign.Right,
         },
         {
-          content: token.total_supply,
+          content: <RoundedBalance value={token.total_supply} ticker={token.symbol} />,
```

Comment:
Actually I prefer this way of formatting the supply: https://github.com/oasisprotocol/explorer/blob/master/src/locales/en/translation.json#L215

With rounded balance:
![image](https://github.com/oasisprotocol/explorer/assets/2093792/6760c6e7-b39c-4719-98e9-e83ab4a65936)


Via i18next: 
![image](https://github.com/oasisprotocol/explorer/assets/2093792/a6204ae0-474d-4251-8ea2-bb6a47f5e421)

Reply:
But I also understand that it is not great with decimals.
I think we need a way that satisfies both criteria.

Can we have the same grouping in RoundedBalance?

(Also, maybe we could consider to resurrect #333?)

Reply:
Ok, I think we are good here. As discussed during meeting, RoundedBalance logic will be adjusted in another PR.